test/e2e/instrumentation/events.go

```diff
@@ -143,7 +143,7 @@ var _ = common.SIGDescribe("Events API", func() {
 		framework.ExpectEqual(foundCreatedEvent, false, "should not have found test event after deletion")
 	})
 
-	ginkgo.It("should delete a collection of events", func() {
+	framework.ConformanceIt("should delete a collection of events", func() {
```

Comment:
If you add it to conformance you need to add a similar comment as in the test above (descirbing what the test is doing).

Reply:
Done.